dev/deps/dependencies-client-spark-3.2

```diff
@@ -20,10 +20,14 @@ commons-crypto/1.0.0//commons-crypto-1.0.0.jar
 commons-io/2.13.0//commons-io-2.13.0.jar
 commons-lang3/3.12.0//commons-lang3-3.12.0.jar
 commons-logging/1.1.3//commons-logging-1.1.3.jar
-failureaccess/1.0.1//failureaccess-1.0.1.jar
-guava/32.1.3-jre//guava-32.1.3-jre.jar
-hadoop-client-api/3.3.6//hadoop-client-api-3.3.6.jar
-hadoop-client-runtime/3.3.6//hadoop-client-runtime-3.3.6.jar
+guava/14.0.1//guava-14.0.1.jar
+hadoop-client-api/3.2.4//hadoop-client-api-3.2.4.jar
```

Comment:
please rebase master and regenerate this files

Reply:
Thank you. I update it.